### PR TITLE
Solution added

### PR DESCRIPTION
**Pull request by Nokafan**

### CODE REVIEW

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -9,6 +16,22 @@
...
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> resultWords = new ArrayList<>();
+        try {
+            List<String> inputLines = Files.readAllLines(Paths.get(fileName));
+            for (String string : inputLines) {
+                for (String oneWord : string.toLowerCase()
+                        .split("[^a-z]+")
+                ) {
+                    if (oneWord.charAt(0) == 'w') {
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/FileWork.java:26]: let's make `w` a constant field

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -9,6 +16,22 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        List<String> resultWords = new ArrayList<>();
+        try {
+            List<String> inputLines = Files.readAllLines(Paths.get(fileName));
+            for (String string : inputLines) {
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/FileWork.java:22]:
```suggestion
            for (String line : inputLines) {
```

File: src/main/java/core/basesyntax/FileWork.java [hunk 3]
```diff
@@ -9,6 +16,22 @@
...
+                for (String oneWord : string.toLowerCase()
+                        .split("[^a-z]+")
+                ) {
+                    if (oneWord.charAt(0) == 'w') {
+                        resultWords.add(oneWord);
+                    }
+                }
+            }
+        } catch (IOException e) {
+            return new String[]{};
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/FileWork.java:32]: let's rethrow a RuntimeException here

File: src/main/java/core/basesyntax/FileWork.java (hunk 2 above)
Comment (Nokafan) [reply src/main/java/core/basesyntax/FileWork.java:22]: Wrong push. I don't have it in my code :(

Comment (Nokafan) [reply src/main/java/core/basesyntax/FileWork.java:22]: Question: is it ok to split declaring
 ` List<String> inputLines = null;
        try {
            inputLines = Files.readAllLines(Paths.get(fileName));
        } catch (IOException e) {`
and put in the try block only one readAllLines?

File: src/main/java/core/basesyntax/FileWork.java (hunk 1 above)
Comment (Nokafan) [reply src/main/java/core/basesyntax/FileWork.java]: Fixed

**Pull request closed without merging**